[Jan 08 17:27:34 2021 | error] asyncio: Task exception was never retrieved
future: <Task finished name='Task-20' coro=<Dispatcher._process_polling_updates() done, defined at C:\Users\Dashedman\AppData\Local\Programs\Python\Python38-32\lib\site-packages\aiogram\dispatcher\dispatcher.py:380> exception=TypeError('not all arguments converted during string formatting')>
Traceback (most recent call last):
  File "C:\Users\Dashedman\AppData\Local\Programs\Python\Python38-32\lib\site-packages\aiogram\dispatcher\dispatcher.py", line 388, in _process_polling_updates
    for responses in itertools.chain.from_iterable(await self.process_updates(updates, fast)):
  File "C:\Users\Dashedman\AppData\Local\Programs\Python\Python38-32\lib\site-packages\aiogram\dispatcher\dispatcher.py", line 225, in process_updates
    return await asyncio.gather(*tasks)
  File "C:\Users\Dashedman\AppData\Local\Programs\Python\Python38-32\lib\site-packages\aiogram\dispatcher\handler.py", line 117, in notify
    response = await handler_obj.handler(*args, **partial_data)
  File "C:\Users\Dashedman\AppData\Local\Programs\Python\Python38-32\lib\site-packages\aiogram\dispatcher\dispatcher.py", line 246, in process_update
    return await self.message_handlers.notify(update.message)
  File "C:\Users\Dashedman\AppData\Local\Programs\Python\Python38-32\lib\site-packages\aiogram\dispatcher\handler.py", line 117, in notify
    response = await handler_obj.handler(*args, **partial_data)
  File "C:\Users\Dashedman\github\cifra_bot\main.py", line 613, in notifications_handler
    keyboard = getFinderKeyboard(expresion, 1, database, dbcursor)
  File "C:\Users\Dashedman\github\cifra_bot\main.py", line 613, in notifications_handler
    keyboard = getFinderKeyboard(expresion, 1, database, dbcursor)
  File "C:\Users\Dashedman\github\cifra_bot\main.py", line 235, in getFinderKeyboard
    cur.execute(f"""
  File "C:\Users\Dashedman\AppData\Local\Programs\Python\Python38-32\lib\site-packages\pymysql\cursors.py", line 161, in execute
    query = self.mogrify(query, args)
  File "C:\Users\Dashedman\AppData\Local\Programs\Python\Python38-32\lib\site-packages\pymysql\cursors.py", line 140, in mogrify
    query = query % self._escape_args(args, conn)
TypeError: not all arguments converted during string formatting
[Jan 08 17:28:38 2021 | info] bot: Shutdown... Please wait!
[Jan 08 17:28:38 2021 | info] aiogram.dispatcher.dispatcher: Stop polling...
[Jan 08 17:28:38 2021 | warning] aiogram: Goodbye!
[Jan 08 17:28:50 2021 | info] bot: Starting...
[Jan 08 17:28:51 2021 | info] aiogram: Bot: Cifra [@cifra_s_bot]
[Jan 08 17:28:52 2021 | warning] aiogram: Updates were skipped successfully.
[Jan 08 17:28:52 2021 | info] bot: Starting demons...
[Jan 08 17:28:52 2021 | info] bot: Listening...
[Jan 08 17:29:10 2021 | info] aiogram.dispatcher.dispatcher: Start polling.
[Jan 08 17:29:11 2021 | error] asyncio: Task exception was never retrieved
future: <Task finished name='Task-19' coro=<Dispatcher._process_polling_updates() done, defined at C:\Users\Dashedman\AppData\Local\Programs\Python\Python38-32\lib\site-packages\aiogram\dispatcher\dispatcher.py:380> exception=AttributeError("'DictCursor' object has no attribute 'morgify'")>
Traceback (most recent call last):
  File "C:\Users\Dashedman\AppData\Local\Programs\Python\Python38-32\lib\site-packages\aiogram\dispatcher\dispatcher.py", line 388, in _process_polling_updates
    for responses in itertools.chain.from_iterable(await self.process_updates(updates, fast)):
  File "C:\Users\Dashedman\AppData\Local\Programs\Python\Python38-32\lib\site-packages\aiogram\dispatcher\dispatcher.py", line 225, in process_updates
    return await asyncio.gather(*tasks)
  File "C:\Users\Dashedman\AppData\Local\Programs\Python\Python38-32\lib\site-packages\aiogram\dispatcher\handler.py", line 117, in notify
    response = await handler_obj.handler(*args, **partial_data)
  File "C:\Users\Dashedman\AppData\Local\Programs\Python\Python38-32\lib\site-packages\aiogram\dispatcher\dispatcher.py", line 246, in process_update
    return await self.message_handlers.notify(update.message)
  File "C:\Users\Dashedman\AppData\Local\Programs\Python\Python38-32\lib\site-packages\aiogram\dispatcher\handler.py", line 117, in notify
    response = await handler_obj.handler(*args, **partial_data)
  File "C:\Users\Dashedman\github\cifra_bot\main.py", line 617, in notifications_handler
    keyboard = getFinderKeyboard(expresion, 1, database, dbcursor)
  File "C:\Users\Dashedman\github\cifra_bot\main.py", line 617, in notifications_handler
    keyboard = getFinderKeyboard(expresion, 1, database, dbcursor)
  File "C:\Users\Dashedman\github\cifra_bot\main.py", line 235, in getFinderKeyboard
    pprint(cur.morgify(f"""
AttributeError: 'DictCursor' object has no attribute 'morgify'
[Jan 08 17:29:22 2021 | info] bot: Shutdown... Please wait!
[Jan 08 17:29:22 2021 | info] aiogram.dispatcher.dispatcher: Stop polling...
[Jan 08 17:29:22 2021 | warning] aiogram: Goodbye!
[Jan 08 17:29:39 2021 | info] bot: Starting...
